pkg/etcdcli/etcdcli.go

```diff
@@ -61,8 +63,16 @@ func (g *etcdClientGetter) getEtcdClient() (*clientv3.Client, error) {
 	}
 	for _, addr := range hostEtcd.Subsets[0].Addresses {
 		if addr.Hostname == "etcd-bootstrap" {
-			// etcd-bootstrap has a valid IP in host-etcd
-			etcdEndpoints = append(etcdEndpoints, fmt.Sprintf("https://%s:2379", addr.IP))
+			// etcd-bootstrap has a valid IP in host-etcd-2
+			hostEtcd2, err := g.endpointsLister.Endpoints(operatorclient.TargetNamespace).Get("host-etcd-2")
+			if err != nil {
+				return nil, err
+			}
+			ip, ok := hostEtcd2.Annotations[BootstrapIPAnnotationKey]
+			if !ok || ip == "" {
```

Comment:
this is optional. If it's not present then you skip it.  Otherwise you will break during upgrades.

Reply:
yeah makes sense